lib/examples/network/model/user.dart

```diff
@@ -13,10 +13,10 @@ class User {
   Map<String, dynamic> toJson() {
     final Map<String, dynamic> data = <String, dynamic>{};
     if (this.data != null) {
-      data['data'] = this.data!.toJson();
+      data['data'] = this.data?.toJson();
```

Comment:
WDYT about using 
<img width="529" alt="image" src="https://user-images.githubusercontent.com/57040392/214893476-7f302a16-4917-475b-8ff4-f379eafbedf4.png">
to avoid confusion with `this.data`